src/FACTFinder/Adapter/Search.php

```diff
@@ -873,4 +915,36 @@ public function getStackTrace()
         $jsonData = $this->getResponseContent();
         return isset($jsonData['stacktrace']) ? $jsonData['stacktrace'] : null;
     }
+    
+    /**
+     * Value for parameter "followSearch" for followups on initial search like filters, pagination, ...
+     * Either from request parameters or from search results "simiFirstRecord".
+     * @return int
+     */
+    public function getFollowSearchValue()
+    {
+        //check if followSearch was set in request data else use simiFirstRecord
+        $searchParameters = FF::getInstance(
+            'Data\SearchParameters',
+            $this->parameters
+        );
+        if($searchParameters->getFollowSearch() !== 10000) {
+            $followSearch =  $searchParameters->getFollowSearch();
+        } else {
+            $jsonData = $this->getResponseContent();
```

Comment:
What is the desired return if no followSearch parameter was set in request data but sorting was?
10000 as Searchparameters returns if no followSearch parameter is set or 0 or null to show followSearch should not be set/used?